tests/helpers.py

```diff
@@ -369,9 +374,12 @@ def get_cephblockpool_name():
     return pool_list
 
 
-def delete_cephblockpool():
+def delete_cephblockpool(cbp_name):
```

Comment:
please verify no one else is using this function as you changed its signature

Reply:
Checked no one else using this function